README.md

```diff
@@ -1 +1,17 @@
-#pnc-js-libs
\ No newline at end of file
+#pnc-js-libs
+
+library at the moment mainly process pnc notifications
+- pnc-dto-types - dto types generated from swagger json file
+- pnc-messagebus-client - code base for messagebus and notification processing
+- umd-bundle - conversion from typescript to javascript
+
```

Comment:
I would add also "##Setup" section with a content like:

"You’ll need to have `Node` and `npm` on your local machine."

So that it's clear no other tools (or Node specific version etc.) are required